dirty_cat/datasets/datasets.py

```diff
@@ -0,0 +1,24 @@
+import pandas as pd
+import os
+
+# TODO: speficy pandas in dependencies
+
+FOLDER_PATH= os.path.dirname(os.path.realpath(__file__))
+print(FOLDER_PATH)
+DATA_NAME = 'midwest_survey.csv'
+
+
+def fetch_data():
+    """
+    function that fetches example data
+    - either from the internet
+    - or locally
+    because the base example dataset (midwest survey) chosen is very small,
+    it is locally stored in data/midwest_survey.csv
+    in the future, for bigger dataset, this function will get the data
+    from a specific url, and store it (optional) before returning it.
+    :return:
```

Comment:
You should abide the numpy doc standard in the docstrings (see http://www.sphinx-doc.org/en/master/ext/napoleon.html#google-vs-numpy ).